app/index.html

```diff
@@ -19,8 +19,8 @@
 
   <link rel="stylesheet" href="../../ms/uiframework/resource/appui/styles/header.css" type="text/css"
   />
-  <link rel="stylesheet" href="/openmrs-standalone/ms/uiframework/resource/uicommons/styles/styleguide/jquery-ui-1.9.2.custom.min.css" type="text/css"/>
-
+  <link rel="stylesheet" href="../../ms/uiframework/resource/uicommons/styles/styleguide/jquery-ui-1.9.2.custom.min.css" type="text/css"/>
+  <link rel="stylesheet" href="../../ms/uiframework/resource/orderentryui/styles/drugOrders.css?cache=1521812153298" type="text/css"/>
```

Comment:
Is this `cache` query param automatically generated? If it is, all is good. Otherwise, if we are trying to prevent the browser from caching the css resource, then it needs to be dynamically generated.

Reply:
Okay, It wasn't autogenerated, I will remove it.